internal/pkg/deploy/cloudformation/stack/validate.go

```diff
@@ -11,7 +11,12 @@ import (
 	"github.com/aws/copilot-cli/internal/pkg/manifest"
 )
 
-// Validation errors when rendering manifest into template.
+// container dependency status constants.
+const (
+	dependsOnStart    = "start"
```

Comment:
What happens if someone puts "START" in their manifest, instead of "start"? Does Cloudformation accept uppercase?

Reply:
I don't think so. Actually im not sure if it would be a good idea to convert all charaters to upper case

Reply:
It seems like the [CF specification](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-ecs-taskdefinition-containerdependency.html#cfn-ecs-taskdefinition-containerdependency-condition) uses all uppercase values. We should convert from arbitrary capitalization in the manifest to `ALLCAPS` when we inject into the template.

Reply:
Ensured all caps on statuses